.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules/
 generator/*.js
 templates/*.js
 api/**/*.js
+api/dist/
```

Comment:
The command below cd into API then outputs back a directory and so isn't dist root level?

Reply:
Good catch, this was a leftover from a previous implementation.